packages/apm/src/integrations/tracing.ts

```diff
@@ -358,31 +347,10 @@ export class Tracing implements Integration {
     });
   }
 
-  /**
-   * Is tracing enabled
-   */
-  private static _isEnabled(): boolean {
-    if (Tracing._enabled !== undefined) {
-      return Tracing._enabled;
-    }
-    // This happens only in test cases where the integration isn't initalized properly
-    // tslint:disable-next-line: strict-type-predicates
-    if (!Tracing.options || typeof Tracing.options.tracesSampleRate !== 'number') {
-      return false;
-    }
-    Tracing._enabled = Math.random() > Tracing.options.tracesSampleRate ? false : true;
-    return Tracing._enabled;
-  }
-
   /**
    * Starts a Transaction waiting for activity idle to finish
    */
   public static startIdleTransaction(name: string, spanContext?: SpanContext): Span | undefined {
-    if (!Tracing._isEnabled()) {
```

Comment:
IIUC from this PR onwards there will be no way to disable Tracing, right?

Reply:
Setting `tracesSampleRate: 0` will disbale sending transactions.
If you do not want the instrumentation, do not install the Integration.

Reply:
:+1: